cli/cmd/doc.go

```diff
@@ -93,3 +111,42 @@ func generateDocs(cmd *cobra.Command) ([]cmdDoc, error) {
 
 	return cmdList, nil
 }
+
+// generateAnnotationsDocs takes a labels file, parse it, iterate over nodes,
+// recognize annotations, adding them to annotationsList.
+func generateAnnotationsDocs(annotationsMap map[string]string, cmdList []cmdDoc) []annotationDoc {
+	var annotationsList []annotationDoc
+
+	for _, cmd := range cmdList {
```

Comment:
There are times when we will add a proxy annotation w/o the corresponding CLI flag. Distributed tracing is an example, where the CLI flag is added later. (That is also why you had to explicitly add the tracing-related descriptions to `GetAnnotationsDocs()` and have a second `for` loop in line 142 below).

I think for simplicity sake,

1. let's move `GetAnnotationsDocs()` to this file since it's not used anywhere else
1. make it an unexported struct, and
1. as @adleong suggested, reuse the CLI commands descriptions in the new struct (I don't mind the duplication being confined in an unexported struct)

Ideally, there should be some sort of struct/map (in a new package under `pkg`) that represents the relationship of the annotation, the CLI option and the description. But since the `proxyConfigOptions` in `cli/cmd/root.go` is unexported and used in many places, modifying it will increase the change scope of this PR.